src/css/components/main.css

```diff
@@ -5,14 +5,27 @@
   display: flex;
   flex-direction: column;
   text-align: center;
-  height: 98%;
-  min-height: 400px;
-  width: 370px;
   margin: auto;
   border: 5px solid var(--main-color);
   border-radius: 3px;
+  width: 28%;
+  min-height: 95%;
+}
+@media (max-width: 1200px) {
+  .App {
+    width: 47%;
+  }
+}
+@media (max-width: 700px) {
+  .App {
+    width: 65%;
+  }
+}
+@media (max-width: 550px) {
+  .App {
+    width: 95%;
```

Comment:
All of these percentages feel like “magic” numbers. Consider moving them into semantically-named CSS variables such as `--tablet-width`